### PR TITLE
Bug Fixes for #42 and #44

### PR DESCRIPTION
**Pull request by parithon**
# TLDR; 

The issues occured when the user opens an individual file and not a folder (workspace). This was because we used relative paths when working with the documents instead of full paths.

## Issue #44

We designed the extension to assume a workspace is always defined. This is not the case when an individual file is opened. When an individual file is opened, the `vscode.workspace.rootPath` would be `undefined` which wouldn't provide the correct fileName used by the `removeHighlighter` to identify which highlight to remove.

## Issue #42

Additionally, because the fileName was inaccurate, the `gotoHighlightCommand` did not have a good reference to open the document. VSCode assumes the incorrect fileName should result in a request for a new document. This is why when you clicked on the highlight tree-item it would open a new document instead of switching to the existing document.

---

# Identified issue

  - The `vscode.workspace.rootPath` is undefined when a user opens an individual file and not a folder (i.e. workspace).

# Resolution

  - Instead of using relative paths we now use full paths. The Treeview Item's label will only show the file name and not the full path.

# Changed Files

## twitchHighlighterTreeview.ts
  - I changed the fileName property to use the full path of the file.
  - I changed the label property to only use the file's name not the full path.
  - I removed the rootPath parameter because it is no longer required.

## extension.ts
  - I changed the `removeHighlighterCommand` to use the fileName parameter without any minipulation for relative paths.
  - I changed the `gotoHighlightCommand` to use the full path of the fileName instead of trying to find the file using a relative path.

# Addresses

#42 
#44

### CODE REVIEW

**Comment by clarkio** [comment]
@parithon thanks for working on this! Would it be possible to also include steps to test? Is it just that we need to open a file without a folder open?

**Comment by parithon** [comment]
@clarkio Correct. If you only had a file open and not a folder the `vscode.workspace.rootPath` would be undefined and our relative paths would be inaccurate.

File: src/extension.ts [hunk 1]
```diff
@@ -523,6 +519,14 @@ export function activate(context: vscode.ExtensionContext) {
     context.subscriptions
   );
 
+  vscode.workspace.onDidCloseTextDocument((document: vscode.TextDocument) => {
```
Comment (parithon) [inline src/extension.ts]: This is used to determine if an untitled document was closed and filter out any highlights associated with it.

Comment (clarkio) [reply src/extension.ts]: Ok cool I understand this now. Thanks!
